tests/src/common/Wsk.scala

```diff
@@ -252,13 +252,13 @@ class WskAction()
      */
     def invoke(
         name: String,
-        parameters: Map[String, JsValue] = Map(),
+        parameters: Map[String, String] = Map(),
```

Comment:
Good idea - I thought there were some already (although they may be in the
java test which are not yet ported to scala).